Makefile

```diff
@@ -0,0 +1,7 @@
+.PHONY: test
+test:
+	nosetests
```

Comment:
I would probably argue that we should be running unit-tests and flake8 for make test, since both ideally will give quick feedback.